subsys/bluetooth/audio/csip_set_member.c

```diff
@@ -736,19 +774,36 @@ static void notify(struct bt_csip_set_member_svc_inst *svc_inst, struct bt_conn
 		   const struct bt_uuid *uuid, const void *data, uint16_t len)
 {
 	int err;
+	const struct bt_gatt_attr *attr;
+
+	k_mutex_lock(&svc_inst_lock, K_FOREVER);
 
-	if (svc_inst->service_p == NULL) {
+	attr = bt_gatt_find_by_uuid(
+		svc_inst->service_p->attrs,
+		svc_inst->service_p->attr_count,
+		uuid);
+
+	if (!attr) {
+		LOG_WRN("Attribute for UUID %p not found", uuid);
+		k_mutex_unlock(&svc_inst_lock);
 		return;
 	}
 
-	err = bt_gatt_notify_uuid(conn, uuid, svc_inst->service_p->attrs, data, len);
+	if (!bt_gatt_is_subscribed(conn, attr, BT_GATT_CCC_NOTIFY)) {
+		LOG_DBG("Connection not subscribed to UUID %p", uuid);
```

Comment:
Suggest to also log the conn pointer 
```suggestion
		LOG_DBG("Connection %p not subscribed to UUID %p", conn, uuid);
```